index.html

```diff
@@ -3,9 +3,256 @@
+        </div>
+
+        <a href="#" class="mobileJoin">Join Cricketing Fusion</a>
+        
+      </div>
+      <a href="#" class="desktopJoin">Join Cricketing Fusion 2023</a>
+    </section>
+
+    <!-- speakers section -->
+    <section class="speakersContainer">
+      <h2>Featured Speakers</h2>
+      <hr class="redLine" />
+
+      <div class="speakersList">
+        <!-- dynamically created section -->
+      </div>
+
+      </div>
+      
+    </section>
+    
+    <section>
+    <div class="page4">
+      <h2 class="partner1">Partner</h2>
+      <hr class="redLine1" />
+      <div class="gridPartner1">
+        <img src="images/partners11.png" />
+      </div>
+    </div>
+    <div class="smallEnd">
+      <div class="img111">
+        <img src="images/copyrightImg.png" />
+      </div>
+      <div class="text1">
+        <h3>2015 Cricket Fusion India<br />Some Rights Reserved</h3>
+      </div>
+    </div>
+  </section>
+
+    <!-- about section for clicked on about tag  -->
+    <section class="aboutPageContainer" >
+      
+      </nav>
+      <!-- page1 of about -->
+      <div class="page1">
+        <div class="humburger">
+          <img
+            src="images/hamburger.png"
+            alt="hamburgerOpen"
+            class="hamburgerOpen1"
+          />
+        </div>
+
+        <div class="homePage1">
+          <h1 class="lineOne1">"Hello! Cricketing World"</h1>
+          <h2 class="lineTwo1">Cricketing Fusion: Uniting Legends 2023</h2>
+          <p class="linePara1">
+            Hey cricket enthusiasts! Get ready for the highly awaited annual
+            Cricket Summit happening in August! Join us in Mumbai, and be a part
+            of the cricketing extravaganza! This year we hope we extend our
+            invitation to organization and indiciduals
+          </p>
+          <div>
+            <p class="lineVenue1">
+              Please contact us per Email for any furthur questions about
+              cricketing Fusion Summit
+            </p>
+            <h3 class="lineData1">CFsummit2023@gmail.com</h3>
+          </div>
+        </div>
+      </div>
+
+      <!-- page2 of about -->
+      <div class="page2">
+        <h2 class="logoHead1">Cricketing Fusion: Uniting Legends 2023</h2>
+        <hr class="redLine1" />
+        <h3 class="logoInfo1">
+          The logo of cricket conference 2023 was decided through the logo
+          competititon on 11 may , 12 may , 13 may
+        </h3>
+        <img src="images/logo.jpeg" class="logoImg1" />
+      </div>
+
+      <!-- page3 of about  -->
+      <div class="page3">
+        <h2 class="logoHead1">See the past Cricket Summit</h2>
+        <hr class="redLine1" />
+        <h3 class="logoInfo1">
+          Take a look at last two CF global Fusion summits took place in
+        </h3>
+        <div class="imgGrid1">
+          <img src="images/img1.jpg" class="img1" />
+          <img src="images/img2.jpg" class="img2" />
+        </div>
+      </div>
```

Comment:
Your design 
![image](https://github.com/bhushan354/capstone-conferencePage/assets/78251927/4fa0759a-0910-4f83-b56b-aa40249b5d9d)

Original design
![image](https://github.com/bhushan354/capstone-conferencePage/assets/78251927/9883f069-c8e7-451b-82b1-ddb2a9e0cd72)

Please let' ensure the elements have similar spacing, `margins nd paddings similar to the original design. 👍